src/components/table/table_row_cell.tsx

```diff
@@ -86,6 +88,7 @@ interface EuiTableRowCellMobileOptionsShape {
    * (typically cells are contained to 50%)
    */
   fullWidth?: boolean;
+  width?: string | number;
```

Comment:
Since this gets applied directly to the style.width, we like to use the `CSSProperties` type. 
```suggestion
  /**
   * Applies the value to the width of the cell in mobile view (typically 50%)
   */
  width?: CSSProperties['width'];
```